nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -253,6 +253,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - `services.avahi.nssmdns` got split into `services.avahi.nssmdns4` and `services.avahi.nssmdns6` which enable the mDNS NSS switch for IPv4 and IPv6 respectively.
   Since most mDNS responders only register IPv4 addresses, most users want to keep the IPv6 support disabled to avoid long timeouts.
 
+- `services.jellyfin` now has jellyfin user gid/uid pinned to 330. It's necessary to manually re-apply permission for files used by jellyfin. ([#292327](https://github.com/NixOS/nixpkgs/pull/292327))
```

Comment:
We should put all text in the release notes especially when the linked PR does not contain a lengthy explanation.